File: src/main/java/com/project/ecoWater/tank/app/TankDTO.java
package com.project.ecoWater.tank.app;

import com.project.ecoWater.sensor.domain.SensorData;
import com.project.ecoWater.user.app.UserDTO;
import com.project.ecoWater.user.domain.User;
import lombok.Builder;
import lombok.Data;

import java.math.BigDecimal;

@Data
@Builder
public class TankDTO {
    private Long tankId;
    private UserDTO user;
    private Float capacity;
    private Float tankHeight;
    private Boolean isMain;
}

File: frontend/Proyecto/Proyecto/app/src/main/java/com/example/proyecto/model/tank/TankDTO.kt
package com.example.proyecto.model.tank

import com.example.proyecto.model.user.UserDTO


data class TankDTO(
    val tankId: Long?=null,
    val tankName: String?=null,
    val capacity: Float?=null,
    val tankHeight: Float?=null,
    val user: UserDTO?=null,
    val isMain: Boolean?=null
)
